### DISCUSSION TITLE
ST75256  270 degrees orientation problem

### DISCUSSION THREAD

**Discussion by FDMRN** [olikraus/u8g2#2266] [Q&A]
I'm using u8g2 with a WO256128 (ST75256) with an 270 degrees orientation (r3) and uigraph_frame_fill does not show the graph unless I execute uigraph_send(0, 0, 256, 128). 
I draw the entire screen again but I can't redraw a specific sector. If I change the orientation it works fine. Any ideas?
Thanks

**Comment by olikraus** [comment]
If updateDisplayArea is used, then this update will only work for R0 out of the box: https://github.com/olikraus/u8g2/wiki/u8g2reference#updatedisplayarea
